### PR TITLE
Rename r_cons_get_ansicon() since it's confusing

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

**Comment by radare** [comment]
I think this function should return bool and be renamed to rcons_is_ansicon() or rcons_is_ansi()

**Comment by kazarmy** [comment]
Ok 20e79e336058df03d9bfc86cd8b8d6b0e9a54d28

File: libr/cons/cons.c [hunk 1]
```diff
@@ -1334,7 +1334,7 @@ R_API int r_cons_get_size(int *rows) {
 
 #if __WINDOWS__
 R_API os_info *r_sys_get_osinfo();
-R_API int r_cons_get_ansicon() {
+R_API bool r_cons_is_ansicon() {
```
Comment (radare) [inline libr/cons/cons.c]: add void int he fucntion definition

Comment (kazarmy) [reply libr/cons/cons.c]: ok 389a2a37dbc73657b6d24619c193fea3063b6c65

**Pull request merged**